src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -1,8 +1,8 @@
 package org.stellar.walletsdk
 
-import org.stellar.sdk.KeyPair
-import org.stellar.sdk.Network
-import org.stellar.sdk.Server
+import org.stellar.sdk.*
+import org.stellar.walletsdk.utils.Operation.Companion.createSponsoredOperation
+import org.stellar.walletsdk.utils.Transaction.Companion.buildTransaction
 
 class Wallet(private val horizonUrl: String, private val networkPassphrase: String) {
```

Comment:
My gut feeling is the Wallet class is going to take a lot more parameters to construct. If that's the case, the [builder pattern ](https://www.baeldung.com/kotlin/builder-pattern) can help to reduce the complexity of object construction.

The [Lombok ](https://projectlombok.org/) library is popular to make the builder pattern less tedious. But there are discussions of deprecating using Lombok with Kotlin. I am not very sure about this. But just bringing this to your attention. 

The current implementation is fine with me.

Reply:
Yes, the Wallet class most likely will have more parameters, but it's hard to tell right now what exactly they will be. Let's optimize when we have more parameters. I'll keep in mind the builder pattern. :pray: